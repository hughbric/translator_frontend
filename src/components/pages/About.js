import React from 'react'

function About() {
  return (
    <div>
      <h1>About</h1>
      <p>In an effort to help extend Health & Safety knowledge to a variety of languages, this is an app that translates terminology.</p>
    </div>
  )
}

export default About;
