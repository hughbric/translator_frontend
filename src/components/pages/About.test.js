import React from "react";
import About from "./About";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("About", () => {
  it("displays about page", () => {
    const wrapper = shallow(<About />);
    const heading = wrapper.find("h1").text();
    expect(heading).toEqual("About");
  });

  it("matches the snapshot", () => {
    const tree = renderer.create(<About />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
