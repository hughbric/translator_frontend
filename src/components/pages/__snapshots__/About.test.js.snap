// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> matches the snapshot 1`] = `
<div>
  <h1>
    About
  </h1>
  <p>
    In an effort to help extend Health & Safety knowledge to a variety of languages, this is an app that translates terminology.
  </p>
</div>
`;
